sos/plugins/yum.py

```diff
@@ -41,6 +41,10 @@ def setup(self):
         # Get a list of channels the machine is subscribed to.
         self.add_cmd_output("yum -C repolist")
```

Comment:
Isn't the non-verbose output redundant, then? This non-verbose variant has the only feature "all collected data of a repo are on one line", like:

    # yum -C repolist
    Updating Subscription Management repositories.
    Last metadata expiration check: 0:10:30 ago on Thu 13 Feb 2020 10:13:56 PM CET.
    repo id                                    repo name                      status
    RedHat_pulp_zoo_repo_pulp_zoo_repo         pulp_zoo_repo                     32
    rhel-8-for-x86_64-appstream-rpms           Red Hat Enterprise Linux 8 for 8,589
    rhel-8-for-x86_64-baseos-rpms              Red Hat Enterprise Linux 8 for 3,676
    satellite-tools-6.6-for-rhel-8-x86_64-rpms Red Hat Satellite Tools 6.6 fo    18
    #

vs. verbose output:

    # yum repolist -C --verbose
    ..
    Repo-id      : RedHat_pulp_zoo_repo_pulp_zoo_repo
    Repo-name    : pulp_zoo_repo
    Repo-revision: 1581322801
    Repo-updated : Mon 10 Feb 2020 09:20:02 AM CET
    Repo-pkgs    : 32
    Repo-size    : 77 k
    Repo-baseurl : https://pmoravec-sat66-on-rhev.gsslab.brq2.redhat.com/pulp/repos/RedHat/Library/custom/pulp_zoo_repo/pulp_zoo_repo
    Repo-expire  : 1 second(s) (last: Thu 13 Feb 2020 10:13:56 PM CET)
    Repo-filename: /etc/yum.repos.d/redhat.repo
    ..
    #

Reply:
There are differences between the commands when you do not have cache on the system.

```
# rm -rf /var/cache/dnf/*
# yum -C repolist
Updating Subscription Management repositories.
repo id                                         repo name
rhel-8-for-x86_64-appstream-rpms                Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
rhel-8-for-x86_64-baseos-rpms                   Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
```

Versus:

```
# rm -rf /var/cache/dnf/*
# yum -C repolist --verbose
Loaded plugins: builddep, changelog, config-manager, copr, debug, debuginfo-install, download, generate_completion_cache, needs-restarting, playground, product-id, repoclosure, repodiff, repograph, repomanage, reposync, subscription-manager, uploadprofile
Updating Subscription Management repositories.
YUM version: 4.2.17
cachedir: /var/cache/dnf
User-Agent: constructed: 'libdnf (Red Hat Enterprise Linux 8.1; generic; Linux.x86_64)'
Error: Cache-only enabled but no cache for 'rhel-8-for-x86_64-baseos-rpms'
```

We can't really just use the `--verbose` flag output in that instance since it will fail on the first repo without cache.